### PR TITLE
bugfix: change code release request

### PR DESCRIPTION
**Pull request by SMoraisAnsys**
Previous request form required to fill a document.

The new approach is to fill an issue at:
https://github.com/ansys-internal/oss-approval-tracklist/issues

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
@SMoraisAnsys We have known issue with vale, in case if it still fails, i think you can ignore the vale.

**Review by MaxJPRey — APPROVED** [review]

File: .github/workflows/ci_cd.yml
[the remark this answers is not in the document]
Comment (SMoraisAnsys) [reply .github/workflows/ci_cd.yml:1]: Yup, works fine 👍

**Comment by SMoraisAnsys** [comment]
@jorgepiloto Any idea why the build documentation fails ? The issues is related to :
```shell
(packaging/structure: line  285) broken    https://www.markdownguide.org/basic-syntax/ - 404 Client Error: Not Found for url: https://www.markdownguide.org/basic-syntax/
```
which seems to be perfectly fine on my side ...

**Comment by jorgepiloto** [comment]
> @jorgepiloto Any idea why the build documentation fails ? The issues is related to :
> 
> ```shell
> (packaging/structure: line  285) broken    https://www.markdownguide.org/basic-syntax/ - 404 Client Error: Not Found for url: https://www.markdownguide.org/basic-syntax/
> ```
> 
> which seems to be perfectly fine on my side ...
> 
> edit : using python `requests` I'm able to reach the website and get a response with code 200 :/

I tried to reproduce locally without success. Everything seems fine. Maybe this is just a temporary issue. Let's wait a bit.

**Comment by PipKat** [comment]
> @jorgepiloto Any idea why the build documentation fails ? The issues is related to :
> 
> ```shell
> (packaging/structure: line  285) broken    https://www.markdownguide.org/basic-syntax/ - 404 Client Error: Not Found for url: https://www.markdownguide.org/basic-syntax/
> ```
> 
> which seems to be perfectly fine on my side ...
> 
> edit : using python `requests` I'm able to reach the website and get a response with code 200 :/

@jorgepiloto Is this perhaps why one of my links that passed the linkcheck yesterday is suddenly failing in your writer's guide PR today?

**Comment by SMoraisAnsys** [comment]
@jorgepiloto @PipKat It is back on track... Just to be sure I also tried to test locally and every thing was fine.
Dunno what caused this error but it seems to be out of our control. Hopefully it was just a temporary problem... :)

**Pull request merged**